### PR TITLE
MINOR: [Website] Add Alenka, Dewey and Rok to commiters

### PR DESCRIPTION
**Pull request by rok**

### CODE REVIEW

File: _data/committers.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@
 - name: Jacques Nadeau
   role: PMC
   alias: jacques
-  affiliation: Dremio
+  affiliation: TDB
```
Comment (rok) [inline _data/committers.yml:73]: @jacques-n

Comment (jacques-n) [reply _data/committers.yml:73]: I'm at Sundeck now.

**Comment by rok** [comment]
@AlenkaF @paleolimbot did I get your details right?

File: _data/committers.yml [hunk 2]
```diff
@@ -292,6 +300,10 @@
   role: Committer
   alias: robertnishihara
   affiliation: Anyscale
+- name: Rok Mihevc
+  role: Commiter
```
Comment (rok) [inline _data/committers.yml:304]:
```suggestion
  role: Committer
```

File: _data/committers.yml (hunk 1 above)
Comment (rok) [reply _data/committers.yml:73]:
```suggestion
  affiliation: Sundeck
```

**Comment by rok** [comment]
> Did you run `scripts/check_committers.py` (I'm not sure it's run on CI)?

I had not. Good call:
```
Missing PMCs in local list: ['houqp']
Missing committers in local list: ['alenka', 'liukun', 'quinnj']
Unexpected members in local list: ['alenkaf', 'liukun4515']
```

**Comment by pitrou** [comment]
Ah, it looks like `alenka` wins against `alenkaf`.

**Comment by rok** [comment]
@pitrou that is just the Apache phonebook!

File: _data/committers.yml [hunk 3]
```diff
@@ -216,6 +228,10 @@
   role: Committer
   alias: ippokratis
   affiliation: Amazon
+- name: Jacob Quinn
+  role: Committer
+  alias: quinnj
+  affiliation: RelationalAI
```
Comment (rok) [inline _data/committers.yml:234]: @quinnj is this right?

Comment (quinnj) [reply _data/committers.yml:234]: Correct!

File: _data/committers.yml [hunk 4]
```diff
@@ -176,6 +180,10 @@
   alias: yibocai
   affiliation: Arm
 
+- name: Alenka Frim
+  role: Committer
+  alias: alenka
```
Comment (AlenkaF) [inline _data/committers.yml:185]:
```suggestion
  alias: alenkaf
```
... if this is the JIRA profile alias, it should be alenkaf 😊

Comment (AlenkaF) [reply _data/committers.yml:185]: Oh, in the phonebook there is `alenka` profile as committer? Oh, what a mess :D

Comment (AlenkaF) [reply _data/committers.yml:185]: Created an INFRA issue, hope they can help =) For now, just leave it as is Rok, thanks!

**Pull request merged**